P4-OVS: Fix shell scripts

nupuruttarwar opened the pull request:
Replace source with '.' in shell scripts to avoid issues when using
Ubuntu or old Bash versions

Title: Fix shell scripts
Change-type: Other
Signed-off-by: Nupur Uttarwar <nupur.uttarwar@intel.com>

mestery commented on install_dep_packages.sh:
```diff
@@ -64,15 +64,12 @@ echo "Number of Parallel threads used: $NUM_THREADS ..."
-elif [[ $OS =~ "Ubuntu" ]]; then
-    sudo apt-get install -y pkg-config
-    sudo apt-get install -y libnl-route-3-dev
+if [ $OS = "Fedora" ]; then
```
Can you also run shellcheck on this? This should likely be something as follows:

```
if [ "$OS" == "Fedora" ]; then
```

The single equal sign assigns the value instead of comparing it.

nupuruttarwar replied:
== is a bash-specific alias for =, which performs a string (lexical) comparison, but changed it to == to be more clear

mestery commented on install_dep_packages.sh:
```diff
@@ -64,15 +64,12 @@ echo "Number of Parallel threads used: $NUM_THREADS ..."
-    sudo apt-get install -y libnl-route-3-dev
+if [ $OS = "Fedora" ]; then
+    sudo dnf install -y pkgconfig libnl3-devel
+elif [ $OS = "Ubuntu" ] then
```
Same here regarding assignment vs. comparison.

nupuruttarwar committed:
P4-OVS: Fix shell scripts

Replace source with '.' in shell scripts to avoid issues when using
Ubuntu or old Bash versions

Title: Fix shell scripts
Change-type: Other
Signed-off-by: Nupur Uttarwar <nupur.uttarwar@intel.com>

nupuruttarwar committed:
Fix shellcheck errors

Signed-off-by: Nupur Uttarwar <nupur.uttarwar@intel.com>
